ocaml/lambda/lambda.ml

```diff
@@ -1666,6 +1679,12 @@ let find_exact_application kind ~arity args =
           if arity <> List.length tupled_args
           then None
           else Some tupled_args
+      (* CR: this allocation is removed by the simplifier and
```

Comment:
Best to put something after the `CR` so we can find it later. Maybe `CR uniqueness`?